pkg/uzip/uzip_test.go

```diff
@@ -58,32 +63,102 @@ func TestFromZip(t *testing.T) {
 		t.Fatalf(`os.ReadFile("testdata/testFolder/subFolder/file4") = _, %v, want nil`, err)
 	}
 
-	if _, err := os.Stat(f1); err != nil {
-		t.Errorf(`os.Stat(%q) =_, %v, want nil`, f1, err)
+	var x []byte
+
+	x, err = ioutil.ReadFile(f1)
+	if err != nil {
+		t.Errorf("open file: %q failed with: %q", f1, err)
+	}
+	if !bytes.Equal(x, f1Expected) {
+		t.Errorf("file %q and file %q are not equal", f1, "testdata/testFolder/file1")
+	}
+	x, err = ioutil.ReadFile(f2)
+	if err != nil {
```

Comment:
sorry about this, but all the error messages need to be made standard. I realize this is new to u-root: 
the old Google internal culture was around this "something: got blah, want bal" format, but the new guidance is for this "x(y) = %v, not %v" form.

Reply:
will do

Reply:
This may take some time until I went through all my PRs to adapt to the new form.